lsb_vsx/build.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 set -e
+set -o pipefail
```

Comment:
It should be probably the part of `lsb_vsx: ensure build script halts on TET, VSX makefile errors` commit, not  `lsb_vsx: supress GCC 14 errors`